cg/apps/tb/api.py

```diff
@@ -147,6 +146,7 @@ def add_pending_analysis(
             "priority": priority,
             "data_analysis": str(data_analysis).upper(),
         }
+        LOG.debug("Submitting job to Trailblazer: %s", request_body)
```

Comment:
Could we rename parameter `type` to `analysis_type` to adress the code factor remark?

Reply:
done